saleor/graphql/csv/mutations/export_products.py

```diff
@@ -94,6 +94,7 @@ def perform_mutation(  # type: ignore[override]
         export_info = cls.get_export_info(input["export_info"])
         file_type = input["file_type"]
 
+        # TODO EEFEE
```

Comment:
What about this? I didn't see changes in this file in previous versions.

Reply:
this is also solved by `get_app_promise`